tests/test-extensions/petstore-extension/src/main/java/org/mule/test/petstore/extension/PetStoreClient.java

```diff
@@ -56,7 +57,7 @@ public int getDisconnectCount() {
     return disconnectCount;
   }
 
-  public boolean isConnected() {
+  public boolean hasActiveConnection() {
```

Comment:
isConnected() suggests that there is a boolean field named connected and that's not the case here.